test/components/__snapshots__/AsyncAPI.test.js.snap

```diff
@@ -0,0 +1,658 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AsyncAPI component should render AsyncAPI v2 document 1`] = `
+"# Streetlights API 1.0.0 documentation
```

Comment:
You can use these snapshots to see the entire rendering in action for both v2 and v3.